### PR TITLE
Update NuGet packages

### PR DESCRIPTION
**Pull request by nopara73**

### CODE REVIEW

File: Stratis.Bitcoin.Features.BlockStore.Tests/Stratis.Bitcoin.Features.BlockStore.Tests.csproj [hunk 1]
```diff
@@ -27,9 +27,9 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="DBreeze" Version="1.85.0" />
+    <PackageReference Include="DBreeze" Version="1.89.0" />
```
Comment (bokobza) [inline Stratis.Bitcoin.Features.BlockStore.Tests/Stratis.Bitcoin.Features.BlockStore.Tests.csproj]: in theory, why not, but have you checked that it works properly?

Comment (nopara73) [reply Stratis.Bitcoin.Features.BlockStore.Tests/Stratis.Bitcoin.Features.BlockStore.Tests.csproj]: Yes, tests pass properly.

File: Stratis.Bitcoin.Features.LightWallet/Stratis.Bitcoin.Features.LightWallet.csproj [hunk 2]
```diff
@@ -17,7 +17,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.Mvc.Core" Version="2.0.0" />
     <PackageReference Include="Microsoft.AspNetCore.Mvc.Versioning" Version="2.0.0" />
-    <PackageReference Include="Serilog.Extensions.Logging.File" Version="1.0.1" />
+    <PackageReference Include="Serilog.Extensions.Logging.File" Version="1.1.0" />
```
[the remark this answers is not in the document]
Comment (nopara73) [reply Stratis.Bitcoin.Features.LightWallet/Stratis.Bitcoin.Features.LightWallet.csproj]: Removed.

**Comment by bokobza** [comment]
Good stuff. I'll wait for after the release to push this in, as DBreeze is used even in the Breeze Wallet for the chain.

**Comment by nopara73** [comment]
Replaced by: https://github.com/stratisproject/StratisBitcoinFullNode/pull/602

**Pull request closed without merging**